static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -253,6 +286,8 @@ function SeeMoreCard({missingMembers, openInviteModal}: SeeMoreCardProps) {
       </Button>
     </MemberCard>
   );
+
+  return <Fragment>{cards}</Fragment>;
```

Comment:
chatted in person

instead of
```
return (<Fragement>{cards}</Fragment>)
```
do
```
return(
  <Fragment>
    {thing.map(() => {
      <CardStuff />
    })}
    <SeeMoreStuff />
  </Fragment>
)
```